src/xmipp/libraries/reconstruction/resolution_monogenic_signal.cpp

```diff
@@ -32,558 +32,306 @@ void ProgMonogenicSignalRes::readParams()
+	addUsageLine("or two half maps.");
+	addUsageLine("Reference: J.L. Vilas et al, MonoRes: Automatic and Accurate Estimation of ");
+	addUsageLine("Local Resolution for Electron Microscopy Maps, Structure, 26, 337-344, (2018).");
+	addUsageLine("  ");
+	addUsageLine("  ");
+	addParamsLine("  --vol <vol_file=\"\">         : Input map to estimate its local resolution map.");
+	addParamsLine("                                : If two half maps are used, it will be the first half map.");
+	addParamsLine("  --minRes <s=30>               : Lowest resolution in (A) for the resolution range");
+	addParamsLine("                                : to be analyzed.");
+	addParamsLine("  --maxRes <s=1>                : Highest resolution in (A) for the resolution range");
+	addParamsLine("                                : to be analyzed.");
+	addParamsLine("  --sampling_rate <s=1>         : Sampling rate (A/px)");
+	addParamsLine("  -o <output_file=\"\">         : Folder where the results will be stored.");
+	addParamsLine("                                : If two half maps are used this should be the first one.");
+	addParamsLine("  [--vol2 <vol_file=\"\">]      : (Optional but recommended) Second half map to estimate its");
+	addParamsLine("                                : local resolution map. The first one will be the --vol label.");
+	addParamsLine("  [--mask <vol_file=\"\">]      : (Optional but recommended) A mask defining the region where ");
+	addParamsLine("                                : the protein is.");
+	addParamsLine("  [--maskExcl <vol_file=\"\">]  : (Optional) This mask excludes the masked region in the ");
+	addParamsLine("                          	   : estimation of the local resolution.");
+
+	addParamsLine("  [--step <s=0.25>]             : (Optional) The resolution is computed from low to high frequency");
+	addParamsLine("  		                       : in steps of this parameter in (A).");
+	addParamsLine("  [--significance <s=0.95>]     : (Optional) The level of confidence for the hypothesis test between");
+	addParamsLine("                                : signal and noise.");
+	addParamsLine("  [--threads <s=4>]             : (Optional) Number of threads to parallelize the algorithm.");
+	addParamsLine("  [--noiseonlyinhalves]         : (Optional) The noise estimation is only performed inside the mask.");
+	addParamsLine("                                : This feature only works when two half maps are provided as input.");
+	addParamsLine("  [--gaussian]                  : (Optional) This flag assumes than the noise is gaussian.");
+	addParamsLine("                                : Usually there are no difference between this assumption and the ");
+	addParamsLine("                                : exact noise distribution. If this flag is not provided, the exact");
+	addParamsLine("                                : distribution is estimated.");
```

Comment:
so what is the point of passing that param? Some significant performance boost?

Reply:
Yes exactly it is significant faster and the  result is quite similar. I added that comment to the documentation